commands/proposal_submission.go

```diff
@@ -1361,9 +1361,8 @@ func checkDataSourceSpec(spec *vegapb.DataSourceDefinition, name string, parentP
 
 				if ethOracle.Trigger != nil &&
 					ethOracle.Trigger.GetTimeTrigger() != nil &&
-					(ethOracle.Trigger.GetTimeTrigger().Initial == nil || *ethOracle.Trigger.GetTimeTrigger().Initial == 0) &&
```

Comment:
Seems reasonable to require an initial time, having the option to specify 'every' without an initial can lean to some arguably unexpected behaviour - e.g if 'every' is set to 1000 and  current block is 999, it will trigger immediately on the next block which I think would be unexpected behaviour, probably better to err on the side of caution and have the initial time as mandatory to avoid any doubt.   Also makes this fix easy as we now have a time against which we can verify eth call events to prevent submission of historical events.

Reply:
what will be the value of every if it's not set here? I don't get how every can be optional and not Initial, I would have expected the inverse.
If initial is not set, set it at time of enactment of the proposal. 
If every is not set, set it to what arbitrary value?

Reply:
Alright I guess, I still think the Initial should be optional and set dynamically to the enactment date of the proposal if not specified, @wwestgarth any thoughts on this whole thing?

Reply:
I feel like setting it based off the enactment data is the more correct thing to do. Making it non-optional is definitely easier, but would need signing off with the spec team.

Reply:
If its set to the enactment date/time of the proposal, then depending on how in/out of sync a nodes etherium client is it may trigger immediately? so I think we would have to then add some offset to prevent this?  seems a bit messy

Reply:
perhaps enactment data plus the every interval?

Reply:
....summary of chat:  The spec activation will always occur sufficiently far ahead of the enactment time for the boundary condition of an eth client's block time being slightly ahead/behind of the vega time resulting in unpredicatable initial trigger behaviour to not be an issue.